app/Http/Controllers/ApiPagesController.php

```diff
@@ -24,22 +24,99 @@ public function index()
 
     /**
      * Display list of apps the user have created and a link to create new appp.
+     * If the user has no app yet, display a page where he can create a new one
      *
      * @return \Illuminate\Http\Response
      */
-    public function myApp()
+    public function showMyApps()
     {
-        return view('api.pages.myapp');
+        if (Auth::check()) {
```

Comment:
**Code Quality - Critical**

Please remove all these unecessary Auth::checks.We have an auth middleware that can be placed on the routes to prevent non-logged in users from accessing such routes and methods.

Please implement this change in other methods where we unecessarily used Auth::check(). Cheers 🐻